packages/babel-plugin-class-features/src/fields.js

```diff
@@ -108,13 +119,24 @@ const privateNameHandlerSpec = {
   get(member) {
     const { classRef, privateNamesMap, file } = this;
     const { name } = member.node.property.id;
-    const { id, static: isStatic } = privateNamesMap.get(name);
+    const {
+      id,
+      static: isStatic,
+      method: isMethod,
+      methodId,
+    } = privateNamesMap.get(name);
 
     if (isStatic) {
       return t.callExpression(
```

Comment:
Also assert here that `// ASSERT: !isMethod` and in some other places where it is needed.